Configurations/10-main.conf

```diff
@@ -1498,12 +1510,14 @@ my %targets = (
     "mingw" => {
         inherit_from     => [ "BASE_unix", asm("x86_asm"),
                               sub { $disabled{shared} ? () : "x86_uplink" } ],
-        cc               => "gcc",
-        cflags           => picker(default => "-m32 -Wall",
+        CC               => "gcc",
+        CFLAGS           => picker(default => "-Wall",
                                    debug   => "-g -O0",
                                    release => "-O3 -fomit-frame-pointer"),
-        cppflags         => combine("-DL_ENDIAN -DWIN32_LEAN_AND_MEAN",
-                                    "-DUNICODE -D_UNICODE", threads("-D_MT")),
+        CPPFLAGS         => "-DWIN32_LEAN_AND_MEAN",
```

Comment:
Mmmmm, I'd say WIN32_LEAN_AND_MEAN is required flag. As far as I recall we have type name conflict with some Windows type or variable and WIN32_LEAN_AND_MEAN masks it.

Reply:
Ah!  Okie, will fix